docs/userGuide/syntax/navBars.mbdf

```diff
@@ -8,10 +8,10 @@ Note: **Navbars** should be placed within a [header file]({{ baseUrl }}/userguid
   </markdown>
 </box>
 
-<include src="outputBox.md" boilerplate >
-<span id="code">
-
-```html
+<!-- bug when using codeAndOutput.md boilerplate, seems to be related to dropdowns -->
```

Comment:
hmm what's the bug here? can't seem to find any differences between `markbind.org` and here

Reply:
Oh. A leftover comment I forgot to remove